pkg/ingester/ingester_test.go

```diff
@@ -822,6 +822,9 @@ func TestIngester_Push(t *testing.T) {
 			assert.Equal(t, int64(expectedTenantsCount), usagestats.GetInt(memoryTenantsStatsName).Value())
 			assert.Equal(t, int64(expectedSamplesCount), usagestats.GetCounter(appendedSamplesStatsName).Total())
 			assert.Equal(t, int64(expectedExemplarsCount), usagestats.GetCounter(appendedExemplarsStatsName).Total())
+			assert.Equal(t, int64(0), usagestats.GetInt(tenantsWithOutOfOrderEnabledStatName).Value())
+			assert.Equal(t, int64(0), usagestats.GetInt(minOutOfOrderTimeWindowSecondsStatName).Value())
+			assert.Equal(t, int64(0), usagestats.GetInt(maxOutOfOrderTimeWindowSecondsStatName).Value())
```

Comment:
Would it be worth it to add a unit test which counts some values in these new metrics?

Reply:
You mean more than https://github.com/grafana/mimir/pull/2940/files#diff-25e63d3b817863ee81ae61ea5bdf7dc4370cdc5033db9066d2afb1c415887d28R5905-R5908 ? Asking just to make sure you haven't missed it.

Reply:
Sorry about that, I did miss those 🤦‍♂️